react-challenge.html

```diff
@@ -40,7 +40,13 @@
 
         render(){
           return(
+            //at first i thought it was a class inheritance issue because thats what the proble says on VS code
 
+            //then looked at the errors in the browser it said that it was missing the opening and closing carrots. I put those on in the VS code and it got rid of the problems tab but it showed even more errors in my browser.
```

Comment:
Great job! Like the Alpha cohort member said at checkout today if you see the errors changing keep going you're doing good stuff!